pkg/test/expectations/expectations.go

```diff
@@ -34,6 +34,18 @@ const (
 	RequestInterval           = 1 * time.Second
 )
 
+func ExpectPodCreated(c client.Client, name string, namespace string) *v1.Pod {
```

Comment:
Created mirrors the call ExpectCreated below, but has a different semantic (get rather than create). WDYT about ExpectPodExists(c, name, namespace). Bonus points if you want to use a v1.NamespacedName